src/components/course/course-header/CourseRunCards.jsx

```diff
@@ -21,18 +22,30 @@ const CourseRunCards = () => {
   const {
     userSubsidyApplicableToCourse,
     missingUserSubsidyReason,
+    applicableCatalogUuid,
```

Comment:
[suggestion] Could `applicableCatalogUuid` be exposed as a `catalogUuid` on the already-returned `userSubsidyApplicableToCourse` value? For example, `userSubsidyApplicableToCourse.catalogUuid`?

There are some subsidy data transforms to a (somewhat) common interface via `getSubsidyToApplyForCourse` where a `catalogUuid` property could be exposed for each subsidy type.